recipes/nmos-cpp/all/conandata.yml

```diff
@@ -1,5 +1,5 @@
 sources:
   # see https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases
-  "cci.20210827":
-    url: "https://github.com/sony/nmos-cpp/archive/14663d595f71d51136ffcd60dd469197bfcbf17d.tar.gz"
-    sha256: "de3fef5bcaefad06c3b5d8452f341701e4c938568672ab6204210a12dd673a17"
+  "cci.20210828":
+    url: "https://github.com/garethsb/nmos-cpp/archive/841b5cd6f5a3378e724d451463cd939bfa605843.tar.gz"
```

Comment:
Waiting for you to merge your last changes

Reply:
Now done 👍